wlauto/common/android/device.py

```diff
@@ -649,6 +649,10 @@ def get_device_model(self):
         except KeyError:
             return None
 
+    def is_wifi_connected(self):
+        self.logger.debug('Checking for wi-fi connectivity.')
+        return self.execute('dumpsys wifi| grep enabled')
```

Comment:
Can this be a bit more intelligent? Looking more into the dump there is curState=ConnectedState.  Does this indicate it has successfully logged in too?  That would be nice to check.